private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -61,8 +59,8 @@ Alternatively, you can omit the input and specify a fully qualified path for the
 type flags struct {
 	ErrorFormat string
 	Type        string
-	Payload     string
-	Output      string
+	From        string
+	To          string
 
 	// special
 	InputHashtag string
```

Comment:
bufcli.BindInputHashtag is still an accurate name because it binds to naked options  `buf beta convert -#format=foo` and seeing that the first argument is `input` it still works